contrib/cf-remote/cf_remote/commands.py

```diff
@@ -136,9 +135,18 @@ def download(tags=None, version=None, edition=None):
         print("No suitable packages found")
     else:
         for artifact in artifacts:
-            download_package(artifact.url)
+            if download:
+                download_package(artifact.url)
+            else:
+                print(artifact.url)
     return 0
 
+def list(tags=None, version=None, edition=None):
+    return _iterate_over_packages(tags, version, edition, False)
```

Comment:
Beware that this hides the `list()` constructor in this file.

Reply:
Changed to `list_command`.